submit_compress.csh

```diff
@@ -12,9 +14,9 @@
 #SBATCH -o %x_%j.errout 
 #SBATCH -e %x_%j.errout 
 # 80 members
-# restarts #SBATCH --ntasks=320 
-# forcing files: 
-#SBATCH --ntasks=405 
+# restarts 
+#SBATCH --ntasks=320 
+# forcing files: #SBATCH --ntasks=405 
```

Comment:
When you get a chance, can you explain to me when it is necessary to switch between 320 and 405 tasks?

Reply:
This is an occasional use script that I tried to restore to its committed form
before committing, but didn't understand yet another subtlety of git.

The choice of the number of tasks depends on whether it will compress
a restart file set, the cpl hist forcing file set, or something else.
I changed that and the date because I wanted to compress 
a restart set so that it was the same size as the others,
and so avoid raising unnecessary flags.  That backfired.

Is lgstarn another git name for Jeff S?